packages/fx-core/src/component/resource/appManifest/utils.ts

```diff
@@ -137,8 +138,11 @@ export class ManifestUtils {
               // inputs[CoreQuestionNames.Features]
               if (inputs.features) {
                 const feature = inputs.features;
-                if (feature === CommandAndResponseOptionItem.id) {
-                  // command and response bot
+                if (
+                  feature === CommandAndResponseOptionItem.id ||
+                  feature == WorkflowOptionItem.id
```

Comment:
Just to confirm. Does workflow bot also implement a `hello world` command as command-response bot does?

Reply:
yes. The workflow bot contains a `helloWorld` command as well as a `doStuff` adaptive card action.